File: src/SMIL.py
line = input()
for i in range(len(line)):
    if line[i] == ":":
        if i < len(line) - 1  and line[i + 1] == ")": print(i)
        elif  i < len(line) - 2 and line[i + 1] == "-"  and line[i + 2] == ")": print(i)
    elif line[i] == ";":
        if i < len(line) - 1 and line[i + 1] == ")": print(i)
        elif i < len(line) - 2 and line[i + 1] == "-" and line[i + 2] == ")": print(i)

File: src/SMIL.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    string line; getline(cin, line);
    for(int i = 0; i < line.length(); i++){
        if(line[i] == ':'){
            if(i < line.length() - 1 && line[i + 1] == ')') cout << i << endl;
        else if(i < line.length() - 2 && line[i + 1] == '-'  && line[i + 2] == ')') cout << i << endl;
        }
        else if(line[i] == ';'){
            if(i < line.length() - 1 && line[i + 1] == ')') cout << i << endl;
        else if(i < line.length() - 2 && line[i + 1] == '-' && line[i + 2] == ')') cout << i << endl;
        }
    }
    return 0;
}